app/Jobs/ProcessTicket.php

```diff
@@ -73,7 +74,7 @@ public function handle()
                 //send push notification to vip
             } elseif ($this->event === 'updated') {
                 Log::info("UPDATED");
-                if ($this->ticket->status === 'done') {
+                if ($this->ticket->status === TicketStatus::Done) {
                     $vipFcmToken = [$user->fcm_token];
                     $title = 'Raccolta VIP eseguita';
```

Comment:
in generale le stringhe mettiamole dentro __() perche cosi possiamo applicare le traduzioni